planning/planning_test_utils/README.md

```diff
@@ -1,92 +1,38 @@
-| motion_velocity_smoother   | NodeTestWithExceptionTrajectory                                                           | trajectory        | trajectory     | Empty, single point, path with duplicate points                                       |
-| obstacle_cruise_planner    | NodeTestWithExceptionTrajectory                                                           | trajectory        | trajectory     | Empty, single point, path with duplicate points                                       |
-| obstacle_stop_planner      | NodeTestWithExceptionTrajectory                                                           | trajectory        | trajectory     | Empty, single point, path with duplicate points                                       |
-| obstacle_velocity_limiter  | NodeTestWithExceptionTrajectory                                                           | trajectory        | trajectory     | Empty, single point, path with duplicate points                                       |
-| obstacle_avoidance_planner | NodeTestWithExceptionTrajectory                                                           | trajectory        | trajectory     | Empty, single point, path with duplicate points                                       |
-| scenario_selector          | NodeTestWithExceptionTrajectoryLaneDrivingMode NodeTestWithExceptionTrajectoryParkingMode | trajectory        | scenario       | Empty, single point, path with duplicate points for scenarios:LANEDRIVING and PARKING |
-| freespace_planner          | NodeTestWithExceptionRoute                                                                | route             | trajectory     | Empty route                                                                           |
-| behavior_path_planner      | NodeTestWithExceptionRoute NodeTestWithOffTrackEgoPose                                    | route             | route odometry | Empty route Off-lane ego-position                                                     |
-| behavior_velocity_planner  | NodeTestWithExceptionPathWithLaneID                                                       | path_with_lane_id | path           | Empty path                                                                            |
-
-## Important Notes
-
-During test execution, when launching a node, parameters are loaded from the parameter file within each package. Therefore, when adding parameters, it is necessary to add the required parameters to the parameter file in the target node package. This is to prevent the node from being unable to launch if there are missing parameters when retrieving them from the parameter file during node launch.
-
-## Future extensions / Unimplemented parts
-
-(WIP)
+The mock data parser is unit tested to ensure there are no defects when parsing the input data.
```

Comment:
@zulfaqar-azmi-t4 
I was listing each package, such as planning_validator, with the aim of implementing tests and increasing coverage. However, if we are removing this, it might be better to remove the entire "Implemented tests" section altogether.

Reply:
@kyoichi-sugahara 
`planning_test_utils` is a library, so it is not the responsibility of the library to list down these things.

Furthermore, it is already in https://autowarefoundation.github.io/autoware.universe/pr-7077/planning/autoware_planning_test_manager/ which makes more sense.

Reply:
@zulfaqar-azmi-t4 
My recommendation is removing chapter of `Implemented tests`. Does it make sense?
If there is motivation to keep following description, you can keep it :+1: 
```
The mock data parser is unit tested to ensure there are no defects when parsing the input data.
```

Reply:
@kyoichi-sugahara 

Yes, that subsection doesn't really contribute to anything.

Removed in https://github.com/autowarefoundation/autoware.universe/pull/7077/commits/36a1fe54c5d2fb4e2be09d1d4507ac4c7b42fa28